### DISCUSSION TITLE
Running into issues when running traefik already on a host and trying to follow new proxy setup

### DISCUSSION THREAD

**Discussion by moekify** [Mailu/Mailu#3644] [User support]
## Environment & Version

- `docker compose version`
- Version: `2024.06.24`

If you are not using docker compose do not file any new issue here.
Kubernetes related issues belong to https://github.com/Mailu/helm-charts/issues
If you are not using docker compose or kubernetes, create a new thread on user support in [disussions](https://github.com/Mailu/Mailu/discussions/categories/user-support).
Non-bug reports (or bug reports that do not follow the template) are moved to [disussions](https://github.com/Mailu/Mailu/discussions).

## Description

I'm trying to setup mailu `2024.06` using a traefik instance that is defined in a different docker-compose file as it is already on my server and handles a lot of other services that are needing certificates etc. supplied via traffic. 

I tried to already change that traefik is using the tls challenge instead of http to retrieve certificates, however, mailu still fails to get certificates, presumably because it cannot reach the acme challenge. 

I saw additional documentation written here: https://github.com/Mailu/Mailu/pull/3440, but think this still doesn't really cover it. It's unfortunately not obvious what has to be done to get it to work. Appreciate any help

**Comment by hifigraz** [comment]
Similar problem here, the mailu traefik documentation could use improvement here.
